xla/stream_executor/gpu/gpu_command_buffer_test.cc

```diff
@@ -1618,6 +1621,105 @@ TEST(GpuCommandBufferTest, ConditionalWhileInExecutionScope) {
   EXPECT_EQ(c_dst, 43);
 }
 
+TEST(GpuCommandBufferTest, CuDnnExplicitConstructionAndUpdateWork) {
+  Platform* platform = GpuPlatform();
+  StreamExecutor* executor = platform->ExecutorForDevice(0).value();
+  TF_ASSERT_OK_AND_ASSIGN(auto stream, executor->CreateStream());
```

Comment:
Please do not use auto here.

Reply:
Done. Note, the repository has ~100 other instances of auto stream ... CreateStream().

Reply:
And the only occurrence of std::unique_ptr<Stream> = ... CreateStream() I found is here: https://github.com/openxla/xla/blob/a3f0b29ab0190dd6ce7ac5b872b3173a4dc0448c/xla/stream_executor/cuda/cuda_timer_test.cc#L73-L81